pkg/alerts/alertsHandler/notificationHandler.go

```diff
@@ -184,11 +184,13 @@ func sendWebhooks(webhookUrl, subject, message string, alertDataMessage string,
 
 	r.Header.Add("Content-Type", "application/json")
 	client := &http.Client{}
-	_, err1 := client.Do(r)
-	if err1 != nil {
+	resp, err := client.Do(r)
+	if err != nil {
 		log.Errorf("sendWebhooks: Error sending request. WebhookURL=%v, Error=%v", webhookUrl, err)
+		return err
 	}
-	return err
+	resp.Body.Close()
+	return nil
```

Comment:
Looks good except for one thing. On lines 181-183, when we get an error we log it but don't return it. With your changes, if we got an error in that block and then got no error on line 187, we'd log an error (line 182), but still return a `nil` error (line 193).

I'm not sure there's a reason we avoided returning an error just after line 182; I think you can return the error there

Reply:
Yes, I just return the `err` after logging it.